.gitignore

```diff
@@ -36,7 +36,7 @@ config_auto.h
 /doc/*.xml
 
 # generated version file
-/src/api/tess_version.h
+/include/tesseract/tess_version.h
```

Comment:
The file name is now always preceded by the `tesseract` directory, so we could simply use `tesseract/version.h`.

Reply:
True